### ISSUE TITLE
2022_08_17_SeedSeq

### ISSUE THREAD

**Issue by shntnu** [broadinstitute/cellpainting-gallery#18]
Segmentation/ Feature extraction has already been performed
Profile creation has already been performed
Data can be public in RODA Immediately

Update as generated:  
[Link to profile repo]  
[Link to publication repo]  
[cellpainting-gallery identifier]  

- [ ] Metadata completely filled out in Project Profiler Database (Imaging Platform internal use only)
- [ ] Segmentation/Feature extraction complete
- [ ] Profiling complete

Transfer to CellPainting Gallery:
- [ ] Upload data to RODA (is private by default)
- [ ] Run validation script to ensure completion
- [ ] Update cellpainting-gallery/README.md
- [ ] Make RODA entry public

If data is being published, prepare for publication:
- [ ] Run Distributed-BioFormats2Raw to create .ome.zarr files
- [ ] Upload (meta)data to IDR (images remain hosted in cellpainting-gallery).

Once published:
- [ ] Make IDR entry public
- [ ] Update cellpainting-gallery/README.md and open-data-registry/cellpainting-gallery.yml to reflect publication
- [ ] Move this Issue from cellpainting-gallery-private to cellpainting-gallery. This step can be performed at an earlier point if it needs inputs from an external collaborator.

**Comment by bethac07** [comment]
Airtable Profiling Experiment tag (and links to other data) is `TA RNAi`. 

The paper only has per-well mean profiles, do we think there is a chance that the raw analysis files and/or backends live anywhere? If so, "In the imaging-platform bucket in the TA folder" is not their location.

**Comment by shntnu** [comment]
> [The paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0131370#sec017) only has per-well mean profiles, do we think there is a chance that the raw analysis files and/or backends live anywhere?

This was the MySQL / BatchProfiler era, so nope :) 

For our reference, this is the project wiki: https://www.ipwiki.app/Z_Paper_on_seed_effects_confounding_RNAi_image-based_profiling (no action needed)

**Comment by bethac07** [comment]
Oh, I bet I know where they are - I bet they are in `s3://imaging-platform-cold/imgdb/imgdb02/2011_07_13_TargetAccelerator-backup.gz`

**Comment by shntnu** [comment]
> Oh, I bet I know where they are - I bet they are in `s3://imaging-platform-cold/imgdb/imgdb02/2011_07_13_TargetAccelerator-backup.gz`

Those will be SQL dumps though

**Comment by bethac07** [comment]
>Those will be SQL dumps though

Yup, but if we want to associate the analysis files, do we have another option for how to get them? I naively assumed we would only want things in the gallery with single-cell analysis (or at least non-normalized per-well analysis), but maybe not?

**Comment by shntnu** [comment]
> Yup, but if we want to associate the analysis files, do we have another option for how to get them? I naively assumed we would only want things in the gallery with single-cell analysis (or at least non-normalized per-well analysis), but maybe not?

Good point. I think we put it in regardless because it will still be useful for the community and, at minimum, will be an incentive to someday get the rest of the (single cell) data. 

e.g. https://github.com/carpenterlab/2017_goldsborough_mlcb/issues/1#issuecomment-1149039103

**Comment by bdiazroh** [comment]
The images are ready to transfer from the imaging-platform bucket, but it seems I don't have permissions to to the transfer, I asked Erin to initiate the transfer with her credentials but she also does't have the permissions.  Once Shantanu gives either one of us the permissions we can upload data to RODA. 
And the other thing is the descriptor of the data set.  Is it cpg0018-RNAi? or cpg0018_RNAi-cellpainting?

**Comment by shntnu** [comment]
We are all set to proceed with the transfer.

@bdiazroh this might still need Beth or Erin to do the transfer. I will discuss in Slack

For my notes: this is in https://github.com/jump-cellpainting/cellpainting-gallery-config/pull/27

The descriptor is `cpg0018-singh-seedseq`

@bdiazroh we maintain a full listing of public and currently embargoed dataset identifiers here https://github.com/broadinstitute/cellpainting-gallery-private#datasets

**Comment by bethac07** [comment]
So to be clear, the final path should be (under the bucket) `cpg0018-singh-seedseq/broad/images/some_batch_name/images/`, with then the folders, right? Any requests for the batch name @shntnu? I can initiate a transfer to that location once confirmed.

**Comment by shntnu** [comment]
> So to be clear, the final path should be (under the bucket) `cpg0018-singh-seedseq/broad/images/some_batch_name/images/`, with then the folders, right? 

Correct - thanks for double-checking 

> Any requests for the batch name @shntnu? I can initiate a transfer to that location once confirmed.

I could not find the creation date, so I'd go with something anodyne: `batch1`

**Comment by bethac07** [comment]
@bdiazroh had already nicely organized everything with a batch name of `Batch1`, so that's how we go :)

Due to permissions disparities, this is going to be a two step sync, but step 1 is underway!

**Comment by bethac07** [comment]
Sync 2 underway!
